app/v4/src/utils/getRandomTestData.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
Why do we need two copies of this file for v3 and v4? Would it be possible to create a util folder at app/ that they can both use?

Reply:
Well they're completely separate projects that just happen to live in the same git repo. By default you can't reference a file outside the project's root folder. Here's an example error if I try to reference a file outside the project root:

> Error: File '/Users/erijiz/Repos/js-e2e-tests/src/utils/getRandomTestData.ts' is not under 'rootDir' '/Users/erijiz/Repos/js-e2e-tests/app/v4'. 'rootDir' is expected to contain all source files.

There's probably some magic settings we could use in the tsconfig.json to make this work, but I don't know how to do it. It's rare for me to have multiple projects in one git repo, which is why I've never been bothered enough to figure it out